src/shared/components/donate/donate.js

```diff
@@ -22,7 +22,7 @@ class Donate extends Component {
             enter the tech industry, and code the future.
           </p>
           <p>Thank you for supporting our mission!</p>
-          <LinkButton text="Donate Now" link="https://donorbox.org/operationcode" />
+          <LinkButton text="Donate Now" link="https://donorbox.org/operationcode" target="_blank" rel="noopener noreferrer" />
```

Comment:
For all of the <LinkButton> changes, we should instead change the actual component:

https://github.com/OperationCode/operationcode_frontend/blob/master/src/shared/components/linkButton/linkButton.js